administration/e2e-tests/003_deleteBayernTests.spec.ts

```diff
@@ -0,0 +1,39 @@
+import { expect, test } from '@playwright/test'
+
+test.describe('DeleteForms', () => {
```

Comment:
i would prefer `DeleteApplications` also for file name. You can also add `Bayern` in the name but for me its not needed